### PR TITLE
Add Langchain4j extension

### PR DESCRIPTION
**Pull request by geoand**

### CODE REVIEW

**Comment by gastaldi** [comment]
Don't forget to add the new team to CODEOWNERS

**Comment by zhfeng** [comment]
Interesting... and do we have `quarkus-langchain4j` repo?

File: terraform-scripts/quarkus-langchain4j.tf [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Create repository
+resource "github_repository" "quarkus_langchain4j" {
+  name                   = "quarkus-langchain4j"
+  description            = "Quarkus Langchain4j extension"
+  archive_on_destroy     = true
+  delete_branch_on_merge = true
+  has_issues             = true
+  vulnerability_alerts   = true
+  topics                 = ["quarkus-extension", "LLM", "AI", "Langchain4j"]
```
Comment (gastaldi) [inline terraform-scripts/quarkus-langchain4j.tf:9]:
```suggestion
  topics                 = ["quarkus-extension", "llm", "ai", "langchain4j"]
```

**Comment by gastaldi** [comment]
> Interesting... and do we have `quarkus-langchain4j` repo?

@zhfeng Merging this PR will create the repository

**Comment by gastaldi** [comment]
> Don't forget to add the new team to CODEOWNERS

See https://github.com/quarkiverse/quarkiverse-devops/pull/188/files for an example

**Comment by geoand** [comment]
Done (I think)

**Review by gastaldi — APPROVED** [review]

**Comment by gastaldi** [comment]
@geoand great job (as usual) 😉

**Comment by geoand** [comment]
I just followed your instructions :).

Thanks again for making this so easy!

**Comment by gastaldi** [comment]
@geoand yes, but be aware that any changes in the repository may be rolledback if they are not present in the Terraform script. You can add yourself as an admin collaborator

**Comment by gastaldi** [comment]
See https://registry.terraform.io/providers/integrations/github/latest/docs/resources/repository_collaborator#example-usage

**Comment by geoand** [comment]
Thanks @gastaldi.

Opened https://github.com/quarkiverse/quarkiverse-devops/pull/191

**Pull request merged**